tap_mysql/__init__.py

```diff
@@ -494,9 +494,13 @@ def get_binlog_streams(mysql_conn, catalog, config, state):
     return resolve_catalog(discovered, binlog_streams)
 
 
-def write_schema_message(catalog_entry, bookmark_properties=[]):
+def write_schema_message(catalog_entry, bookmark_properties=[], add_deleted_field=False):
```

Comment:
Don't you need to call from somewhere with `add_deleted=True`?

Reply:
oh yes.